components/prism-php.js

```diff
@@ -42,14 +49,19 @@
 		}
 	});
 
-	Prism.languages.insertBefore('php', 'keyword', {
+	// Must be defined before class-name (because primitive types must be matched before class names in function typehint)
+	Prism.languages.insertBefore('php', 'class-name', {
 		'variable': /\$+(?:\w+\b|(?={))/i,
 		'package': {
-			pattern: /(\\|namespace\s+|use\s+)[\w\\]+/,
+			pattern: /(\\|namespace\s+|use\s+(function\s+)*)[a-zA-Z_][\w\\]+/,
 			lookbehind: true,
 			inside: {
 				punctuation: /\\/
 			}
+		},
+		'type': {
+			pattern: /(bool|boolean|int|integer|float|string|object|void|array|mixed)(?!_)/,
+			greedy: !0
```

Comment:
Same for all the others.

```suggestion
			greedy: true
```

Reply:
copy pasted from minified code -> fixed in next commit